### PR TITLE
Adding QA redirects for project page requests.

### PR DESCRIPTION
**Pull request by weerd**
This PR adds an update to send all `*/project/*` URLs that previously went somewhere in Ashes over to Phoenix.

Refs [163236109](https://www.pivotaltracker.com/story/show/163236109)

### CODE REVIEW

**Comment by DFurnes** [comment]
Rebased against `master` to get latest changes that have been recently merged in.

File: dosomething-qa/fastly-frontend/projectpages_recv.vcl [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Catch any */project/* page requests.
+if (req.url ~ "(?i)^\/((us|mx|br)\/)?project\/")
+  error 779 "Not Found";
```
Comment (DFurnes) [inline dosomething-qa/fastly-frontend/projectpages_recv.vcl]: Whoop, these have a syntax error in them:

```
* fastly_service_v1.frontend-qa: [ERR] Invalid configuration for Fastly Service (5I35is1gX0tbds1VfNmBIb): Syntax error: Expected `{`, got `error`
at: (input Line 657 Pos 3)
  error 779 "Not Found";
--#####-----------------
```

We need to add an opening `{` on the if conditional above!

Comment (weerd) [reply dosomething-qa/fastly-frontend/projectpages_recv.vcl]: doh!

File: dosomething-qa/fastly-frontend/projectpages_error.vcl [hunk 2]
```diff
@@ -0,0 +1,6 @@
+if (obj.status == 779) {
+  # Redirect any requests to */project/* and send them to the homepage.
+  set obj.http.Location = "/us"
```
Comment (DFurnes) [inline dosomething-qa/fastly-frontend/projectpages_error.vcl]: Oops, another sneaky syntax error here:

```
* module.dosomething-qa.module.fastly-frontend.fastly_service_v1.frontend-qa: 1 error(s) occurred:

* fastly_service_v1.frontend-qa: [ERR] Invalid configuration for Fastly Service (5I35is1gX0tbds1VfNmBIb): Syntax error: Expected variable, string or semicolon got `set`
at: (input Line 1079 Pos 3)
  set obj.status = 302;
--###------------------
```

We need a semi-colon on the line above here (after `"/us"`).

**Comment by DFurnes** [comment]
Applied & confirmed `/project/` paths are now redirecting to the homepage! 👍

**Pull request merged**